Home_Task_13_1.py

```diff
@@ -0,0 +1,121 @@
+class MyCard:
+
+    def __init__(self, card_num, name, surname, expare_month, expare_year, cvv, pin, balance=0):
+        self.set_card_num(card_num)
+        self.set_person(name, surname)
+        self.set_expare_month(expare_month)
+        self.set_expare_year(expare_year)
+        self.set_cvv(cvv)
+        self.set_pin(pin)
+        self.balance = balance
+
+    def set_card_num(self, card_num):
+        if self.valid_number(card_num):
+            self.card_num = card_num
+        else:
+            raise ValueError("The card must be a 16-digit number.")
+
+    def get_card_num(self):
+        return self.card_num
+
+    @staticmethod
+    def valid_number(card_num):
+        return isinstance(card_num, int) and len(str(card_num)) == 16
```

Comment:
"16" <- such hardcodes better to place as class attributes that will automatically modify static method to classmethod and you will avoid hardcodes